pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -516,7 +518,11 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 			if err != nil {
 				return fmt.Errorf("error getting list of volumes to backup: %v", err)
 			}
-
+			kdmpData, err := core.Instance().GetConfigMap(drivers.KdmpConfigmapName, drivers.KdmpConfigmapNamespace)
+			if err != nil {
+				return fmt.Errorf("error readig kdmp config map: %v", err)
```

Comment:
I think it is better to log the message and continue with the default option for the backup rt?